docs/javascript/v2/features/simulcast.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Simulcast
+nav: 3.33
+---
+
+## Minimum requirements
+
+- SDK versions `@100mslive/hms-video-store@0.8.0`, `@100mslive/react-sdk@0.6.0` or later
+- Simulcast to be enabled from dashboard template
+
+Simulcast is a technique by which a WebRTC client(publisher) encodes the same video stream multipe times in different resolutions and bitrates and sends these to the server(SFU) who then decides who(subscriber) receives which of the published resolutions.
+
+To turn on simulcast in your room, open the [templates](https://dashboard.100ms.live/templates) in the dashboard and enable it for roles there.
```

Comment:
i think this feature needs more info, and i've added that in #921. do you think this section is required in this doc - given that it focuses on subscriber-side code?

Reply:
If it is already covered we can just link to that.